app/javascript/components/rooms/FeatureTabs.jsx

```diff
@@ -10,7 +10,7 @@ export default function FeatureTabs() {
     <Row className="py-5">
```

Comment:
This row seems to be messing the layout for some reason - the FeatureTab is expanding over the container width.
Adding w-100 (width: 100%) to className will fix the overflow on the right (but not on the left somehow).